cmd/ttn-lw-cli/commands/config.go

```diff
@@ -20,11 +20,21 @@ import (
 	"go.thethings.network/lorawan-stack/cmd/internal/commands"
 	conf "go.thethings.network/lorawan-stack/pkg/config"
 	"go.thethings.network/lorawan-stack/pkg/log"
+	"go.thethings.network/lorawan-stack/pkg/rpcmiddleware/discover"
 )
 
+func httpAddress(host string, tls bool) string {
+	if tls {
+		return fmt.Sprintf("https://%s:8885", host)
+	}
+	return fmt.Sprintf("http://%s:1885", host)
+}
+
```

Comment:
This should probably be extracted in a separate package, just like `discover.WithDefaultPort`

Reply:
Why don't we just use the same logic for HTTP as we do with GRPC? So actually using `discover.WithDefaultPort`?

Reply:
This should definitely be in the discovery package

Reply:
For HTTP, we also need the protocol (http/https). Added a `discover.DefaultURL` and using that.